demo/scrollable-page.html

```diff
@@ -56,14 +56,20 @@
 </head>
 <body>
 <div class="scrollmargin">
-    <span onclick="scroll()" class="large-button">
+    <span onclick="scroll()" class="large-button" onload="return false;">
+      👇🏻
+      🗑
```

Comment:
This should be an elephant 

```suggestion
      🐘
```